test/cql-pytest/cassandra_tests/functions/operation_fcts_test.py

```diff
@@ -0,0 +1,819 @@
+            row("जॉन जॉन", "जॉन Doe", "Doe जॉन", "Doe Doe"))
+
+@pytest.mark.xfail(reason="#2693")
+def testSingleOperations(cql, test_keyspace):
+    with create_table(cql, test_keyspace, "(a tinyint, b smallint, c int, d bigint, e float, f double, g varint, h decimal, PRIMARY KEY(a, b, c))") as table:
+        execute(cql, table, "INSERT INTO %s (a, b, c, d, e, f, g, h) VALUES (1, 2, 3, 4, 5.5, 6.5, 7, 8.5)")
+
+        # Test additions
+        with original_column_names(cql) as cql:
+            assertColumnNames(execute(cql, table, "SELECT a + a, b + a, c + a, d + a, e + a, f + a, g + a, h + a FROM %s WHERE a = 1 AND b = 2 AND c = 1 + 2"),
+                          "a + a", "b + a", "c + a", "d + a", "e + a", "f + a", "g + a", "h + a")
+
+        assertRows(execute(cql, table, "SELECT a + a, b + a, c + a, d + a, e + a, f + a, g + a, h + a FROM %s WHERE a = 1 AND b = 2 AND c = 1 + 2"),
+                   row(2, 3, 4, 5, 6.5, 7.5, 8, Decimal("9.5")))
+
+        assertRows(execute(cql, table, "SELECT a + b, b + b, c + b, d + b, e + b, f + b, g + b, h + b FROM %s WHERE a = 1 AND b = 2 AND c = 1 + 2"),
+                   row(3, 4, 5, 6, c_float(7.5).value, 8.5, 9, Decimal("10.5")))
+
+        assertRows(execute(cql, table, "SELECT a + c, b + c, c + c, d + c, e + c, f + c, g + c, h + c FROM %s WHERE a = 1 AND b = 2 AND c = 1 + 2"),
+                   row(4, 5, 6, 7, c_float(8.5).value, 9.5, 10, Decimal("11.5")))
+
+        assertRows(execute(cql, table, "SELECT a + d, b + d, c + d, d + d, e + d, f + d, g + d, h + d FROM %s WHERE a = 1 AND b = 2 AND c = 1 + 2"),
+                   row(5, 6, 7, 8, 9.5, 10.5, 11, Decimal("12.5")))
+
+        assertRows(execute(cql, table, "SELECT a + e, b + e, c + e, d + e, e + e, f + e, g + e, h + e FROM %s WHERE a = 1 AND b = 2 AND c = 1 + 2"),
+                   row(c_float(6.5).value, c_float(7.5).value, c_float(8.5).value, 9.5, c_float(11.0).value, 12.0, Decimal("12.5"), Decimal("14.0")))
+
+        assertRows(execute(cql, table, "SELECT a + f, b + f, c + f, d + f, e + f, f + f, g + f, h + f FROM %s WHERE a = 1 AND b = 2 AND c = 1 + 2"),
+                   row(7.5, 8.5, 9.5, 10.5, 12.0, 13.0, Decimal("13.5"), Decimal("15.0")))
+
+        assertRows(execute(cql, table, "SELECT a + g, b + g, c + g, d + g, e + g, f + g, g + g, h + g FROM %s WHERE a = 1 AND b = 2 AND c = 1 + 2"),
+                   row(8, 9, 10, 11, Decimal("12.5"), Decimal("13.5"), 14, Decimal("15.5")))
+
+        assertRows(execute(cql, table, "SELECT a + h, b + h, c + h, d + h, e + h, f + h, g + h, h + h FROM %s WHERE a = 1 AND b = 2 AND c = 1 + 2"),
+                   row(Decimal("9.5"),
+                       Decimal("10.5"),
+                       Decimal("11.5"),
+                       Decimal("12.5"),
+                       Decimal("14.0"),
+                       Decimal("15.0"),
+                       Decimal("15.5"),
+                       Decimal("17.0")))
+
+        # Test substractions
```

Comment:
```suggestion
        # Test subtractions
```

Reply:
Also in https://github.com/scylladb/scylladb/pull/20502/files#diff-45e25e9ead740dfdf0890e01697b50101fb6e77d3d5ebbd2ea8e60e008e90b0bR330

Reply:
As I explained, I didn't write this test from scratch, and rather translated it from Java code. The translation is not automatic but I do try to avoid unnecessary changes. In particular, I don't fix comments - just change the "//" to "#" (as you saw in one place I left both, this was a mistake), and don't change names of functions so their style isn't always our own favorite style.
Because I keep all these translated tests in one subdirectory - test/cql-pytest/cassandra_tests - it doesn't hurt anyone that their style is different. No Scylla developer should ever add their own tests to the files in this directory.